tests/ut/ir/transforms/test_auto_tile_vector.py

```
@@ -0,0 +1,2227 @@
+def test_ragged_and_half_width_pointwise_are_admitted(program, op):
+    structure = _structure(_run_auto_tile(program))
+    assert structure.spmd == 1
+    assert structure.ops[op] >= 1
+    assert structure.ops["tensor.assemble"] >= 1
+
+
+@pytest.mark.parametrize("program", [Bf16PointwiseProgram, Bf16RowReductionProgram])
+def test_bf16_arithmetic_is_rejected_during_910b_admission(program):
+    with pytest.raises(
+        ValueError,
+        match="Ascend910B vector arithmetic supports FP16 and FP32.*BF16 tensors are supported only",
```

Ruff (0.16.1)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)